src/DebugConfigurationProvider.ts

```diff
@@ -168,6 +168,9 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
     if (cmd === 'yarn') {
       return { runtimeExecutable: 'yarn', args: [], ...commonConfig };
     }
+    if (cmd === 'pnpm') {
+      return { runtimeExecutable: 'pnpm', args: [], ...commonConfig };
+    }
```

Comment:
If they are identical (other than the cmd itself), maybe combine the if condition. Just curious if pnpm requires extra "--" like npm when passing extra arguments?